src/TwigComponent/CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # CHANGELOG
 
+## 2.25.1
+
+-   [SECURITY] `ComponentAttributes` now requires an `EscaperRuntime` as second argument
```

Comment:
```suggestion
-  [SECURITY] `ComponentAttributes` now requires a `Twig\Runtime\EscaperRuntime` as second argument
```

Reply:
Nope for the space... or at least "not the one used in this file, exccept for 2.20"

Reply:
Updated message to add fqcn thx!